generate-profile.lic

```diff
@@ -0,0 +1,120 @@
+=begin
+This script takes partial YAML from profile-gen/ and resolves all %INCLUDE
+directives to produce new YAML files in profiles/ .
```

Comment:
Now, if we could just wire this into `dependency` and have it run at settings-load time this could all be transparent to the user!

Reply:
I like this idea, but it's probably too expensive to call this every time we run get_settings unless functionality is added to limit the scope (which could be done).  E.g. if `;generate-profile foo` only parsed the deps for Name-foo.yaml and Name-setup.yaml .

Reply:
Agreed. I'm just making a mental note of what I want for Christmas next year.